jupyter-notebooks/subscriptions_api_tutorial/LST_PV_Sub_Phoenix_AZ.ipynb

```diff
@@ -0,0 +1,660 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Extract the date from the \"item_datetime\" column\n",
+    "df[\"date\"] = pd.to_datetime(df[\"item_datetime\"].dt.date)\n",
+    "\n",
+    "# Filter for LST values over 335K\n",
+    "high_temp_df = df[df[\"lst.band-1.mean\"] > 335]\n",
+    "\n",
+    "# Group the data by date and count the frequency of the number of days with LST over 335K\n",
+    "freq_data = high_temp_df.groupby(\"date\").size()\n",
+    "\n",
+    "# Create the plot\n",
+    "fig, ax = plt.subplots(figsize=(10, 6))\n",
+    "freq_data.plot(kind='bar', ax=ax)\n",
+    "ax.set_xlabel(\"Date\")\n",
+    "ax.set_ylabel(\"Frequency\")\n",
+    "ax.set_title(\"Frequency of Days with LST over 335K in Phoenix, AZ\")\n",
+    "date_labels = freq_data.index.strftime(\"%Y-%m-%d\")\n",
+    "ax.set_xticklabels(date_labels)\n",
+    "\n",
+    "plt.show()\n"
```

Comment:
I'm not sure this plot is necessary here, especially since we have so few data points for this example. I don't think a bar chart makes sense because each day can only have a frequency of 1 (1 day with LST over 335 per day seems a bit redundant).

Are we just trying to illustrate that for these days, the LST is above 335K? If we have more data points, maybe we can do a line plot with the date still on the x-axis, and a binary above threshold/below threshold on the y-axis.

Otherwise, for just a few data points like we have here, I think it's okay to just show a chart of the values to illustrate that for these days, they are all above our set threshold.

What do you think?

Reply:
Agreed. Deleted.